src/item/enchantment/ItemFlags.php

```diff
@@ -29,27 +29,30 @@ private function __construct(){
 		//NOOP
 	}
 
-	//TODO: this should probably move to protocol
-
 	public const NONE = 0x0;
-	public const ALL = 0xffff;
+	public const ALL = 0xfffff;
+	public const BREAKABLE = self::ALL & ~self::COMPASS & ~self::HEAD_MASK;
 	public const ARMOR = self::HEAD | self::TORSO | self::LEGS | self::FEET;
 	public const HEAD = 0x1;
 	public const TORSO = 0x2;
 	public const LEGS = 0x4;
 	public const FEET = 0x8;
 	public const SWORD = 0x10;
 	public const BOW = 0x20;
-	public const TOOL = self::HOE | self::SHEARS | self::FLINT_AND_STEEL;
+	public const TOOL = self::SHEARS | self::FLINT_AND_STEEL;
```

Comment:
HOE still a tool

Reply:
I'm pretty sure these values originally came from vanilla directly, so I would hesitate to modify them unless it's provably incorrect.